integration/airflow/README.md

```diff
@@ -41,25 +41,14 @@ To install from source, run:
 $ python3 setup.py install
 ```
 
-## Usage
+## Setup
 
-### Airflow 1.10+
-
-To begin collecting Airflow DAG metadata with OpenLineage, use:
-
-```diff
-- from airflow import DAG
-+ from openlineage.airflow import DAG
-```
+### Airflow 2.3+
 
-When enabled, the library will:
-
-1. On DAG **start**, collect metadata for each task using an `Extractor` if it exists for given operator.
-2. Collect task input / output metadata (`source`, `schema`, etc)
-3. Collect task run-level metadata (execution time, state, parameters, etc)
-4. On DAG **complete**, also mark the task as _complete_ in OpenLineage
+Integration automatically registers itself for Airflow 2.3 if it's installed on Airflow worker's python.
+This means you don't have to do anything besides configuring it, which is described in Configuration section.
 
-### Airflow 2.1+ (*experimental*)
+### Airflow 2.1 - 2.2
```

Comment:
_Minor_: We'll eventually want to discourage users from using the `LineageBackend` as it's not actually supported and has limitations.

Reply:
We want to, but it's the only stuff that's working below 2.3. Maybe add some additional disclaimer?

Reply:
Great, thanks for adding the disclaimer 👍